pomodoro-cli.sh

```diff
@@ -10,28 +10,48 @@ function notify() {
   header=$1
   body=$2
   if [[ $kernel == "Darwin" ]]; then
-	osascript -e "display notification \"${body:?}\" with title \"${header:?}\""
+	osascript -e "display notification \"${body:?}\" with title \"${header:?}\" sound name \"~/.config/pomodoro-cli/sound.wav\""
   else
     notify-send -u critical -t 0 -a pomodoro-cli "${header:?}" "${body:?}"
   fi
 }
 
 ### Show a countdown timer and a message and updates without
-### cleaning the whole screen
-## Params: seconds {Number} - Countdown time in seconds 
+### cleaning the whole screen. Also kills apps if needed
+## Params: seconds {Number} - Countdown time in seconds
+## Params: is_focus_time {String} - "FOCUS" or "BREAK" or "LONG_BREAK"
+## Params: app_names {String} - Comma separated list of apps to kill
 ## Params: message {String} - Timer description
+## Item: IFS=',' is (Internal Field Separator) set to comma
 ## Returns: {String} - "hh:mm:ss - $message"
 function countdown(){
   secs=$1
+  is_focus_time=$2
+  app_names=$3
+  message=$4
+
+  IFS=','
+  read -ra arr <<< "$app_names"
+
   shift
-  message=$*
+  
   while [ "$secs" -gt -1 ]
   do
+    if [[ $is_focus_time == "FOCUS" ]]; then
+      for app_name in "${arr[@]}"; do
+        pkill --signal STOP "$app_name"
+      done
+    fi
```

Comment:
<img width="518" alt="image" src="https://user-images.githubusercontent.com/86282911/206885554-80bb2fb5-1ae7-41ef-9790-9076c33e0a45.png">
won't work in macos